src/hexen/p_enemy.c

```diff
@@ -528,25 +528,29 @@ boolean P_LookForPlayers(mobj_t * actor, boolean allaround)
     player_t *player;
     angle_t an;
     fixed_t dist;
+    int i = 0; // For breaking vanilla infinite loop.
 
     if (!netgame && players[0].health <= 0)
     {                           // Single player game and player is dead, look for monsters
         return (P_LookForMonsters(actor));
     }
     c = 0;
 
-    // NOTE: This behavior has been changed from the Vanilla behavior, where
```

Comment:
There should be a warning comment here that `3` can't be changed to `maxplayers` or desyncs will result.